common/v2/services/EthService/ens/validators.ts

```diff
@@ -26,7 +26,7 @@ export function getValidTLDsForChain(chainId: number): ITLDCollection {
 export function isValidENSName(str: string) {
   try {
     return (
-      str.length > 6 && !str.includes('.') && normalise(str) !== '' && str.substring(0, 2) !== '0x'
+      str.length >= 3 && str.includes('.') && normalise(str) !== '' && str.substring(0, 2) !== '0x'
```

Comment:
- Is there a specific reason to choose a length of 3?
- Can you add a unit test so we can document the intended behaviour.

Reply:
The minimum amount of characters for an ENS name is 3 (https://medium.com/the-ethereum-name-service/announcing-the-ens-3-6-character-auction-31bfee7a4eab).

I'll add a test unit 👍